app/components/search/SearchResults/SearchResults.module.scss

```diff
@@ -1,85 +1,136 @@
-@import '../../../styles/utils/_helpers.scss';
+@import '~styles/utils/_helpers.scss';
 
-.searchResultsContainer {
+.searchMapContainer {
   position: relative;
+  display: flex;
+  flex-direction: row;
+  @media screen and (max-width: $break-tablet-s) {
+    flex-flow: column-reverse;
+  }
```

Comment:
I don't know how comfortable you are with CSS grid, but I think it would be a better fit for the layout we're trying to accomplish here than flexbox. For single-dimensional layouts like this, we still benefit from using CSS grid because it allows the parent element be responsible for positioning and sizes of its children, and then the children don't have to worry about it.

If you're not familiar with CSS Grid, I've found the [CSS Tricks page on grid](https://css-tricks.com/snippets/css/complete-guide-grid/) to be a super useful resource.

I think you could make it look something like this:

```css
.searchMapContainer {
  display: grid;

  /* Large layout */
  grid-template-columns: 6fr 4fr;  /* Replaces the width: 60% on .searchResultsContainer */
  grid-areas: "results map";
  grid-column-gap: 15px;  /* Replaces the margin-right: 15px on .searchResultsContainer */

  /* Phone layout */
  @media screen and (max-width: $break-tablet-s) {
    grid-template-columns: auto;
    grid-template-rows: auto auto;
    grid-areas:
      "map"
      "results";
  }
}

.searchResultsContainer {
  grid-area: "results";
}

.searchMap {
  grid-area: "map";
}
```

Reply:
That's a great bit of magic. I've only scratched the surface of grid thus far. Your sketch worked more or less out of the box and removed several lines of CSS